tests/Behat/AuthenticationContext.php

```diff
@@ -115,6 +115,28 @@ public function visitOwnUserPage(): void {
     $this->visitPath($url->getInternalPath());
   }
 
+  /**
+   * Navigates to other user's page.
+   *
+   * @var string $username
+   *   The name of the user whose page is to be visited.
+   *
+   * @Given I visit the :user_name user page
+   *
+   * @throws \Exception
+   *   Thrown when the user with the given name does not exist.
+   */
+  public function visitUserPage(string $user_name): void {
+    /** @var \Drupal\user\Entity\User $user */
```

Comment:
This would include FALSE at this step in the method.
Only after the `if (..)` the FALSE goes away.
```suggestion
    /** @var \Drupal\user\Entity\User|false $user */
```
However, it seems we don't really need this at all.
The `user_load_by_name()` is documented to return `UserInterface|false`, which is good enough for everything we do with that variable.

Reply:
so
```suggestion
```